src/automatic_differentiation.jl

```diff
@@ -433,19 +433,37 @@ curl(f::F, v::Vec{2, T}) where {F, T} = curl(f, Vec{3}((v[1], v[2], T(0))))
 """
     laplace(f, x)
 
-Calculate the laplacian of the scalar field `f`, in the point `x`.
+Calculate the laplacian of the field `f`, in the point `x`.
+If `f` is a vector field, use broadcasting.
 
 # Example
 ```jldoctest
-julia> f(x) = norm(x);
-
 julia> x = rand(Vec{3});
 
+julia> f(x) = norm(x);
+
 julia> laplace(f, x)
 1.7833701103136868
+
+julia> g(x) = x*norm(x);
+
+julia> laplace.(g, x)
+3-element Tensors.Tensor{1,3,Float64,3}:
+ 2.10739
+ 2.73497
+ 2.01962
 ```
 """
 function laplace(f::F, v) where F
     return div(x -> gradient(f, x), v)
 end
 const Δ = laplace
+
+function Base.broadcast(::typeof(laplace), f::F, v::Vec{3}) where {F}
+    @inbounds begin
+        v1 = laplace(x->f(x)[1], v)
```

Comment:
Isn't there a better way than calling the function 3 times?

Reply:
Probably.